examples/MongoDB-Atlas-Third-Party-Integration/Readme.md

```diff
@@ -0,0 +1,108 @@
+# Example - A basic example to start with the MongoDB Atlas and Terraform
```

Comment:
The title is not correct for what it adds.  This is a base install along with an example of using 3rd party integrations, Teams and Prometheus.  We should re-title and add it to the list below on lines 5-8.  

We should discuss how to best do examples.  For intergrations you don't have to have a cluster, db user, access list to setup the integrations so we could make it a small targeted example and likely make it a bit easier to grok.

Reply:
Corrected description simplified example

Reply:
ty!  Really nice